.github/workflows/git-artifacts.yml

```diff
@@ -0,0 +1,316 @@
+name: git-artifacts
+
+on:
+  # Uncomment the "push:" line if you want to build artifacts without sending
+  # a `repository_dispatch` event, and possibly set the environment variables
+  # `BUILD_ONLY`, `REPOSITORY` and `REF` explicitly.
+  #
+  # push:
+  repository_dispatch:
+    types:
+      - git-artifacts
+
+env:
+  GIT_CONFIG_PARAMETERS: "'user.name=${{github.actor}}' 'user.email=${{github.actor}}@users.noreply.github.com' 'user.signingkey=mine'"
+  GPG_OPTIONS: "--batch --yes --no-tty --list-options no-show-photos --verify-options no-show-photos --pinentry-mode loopback"
+  HOME: "${{github.workspace}}\\home"
+  MSYSTEM: MINGW64
+  PACKAGER: ${{github.actor}} <${{github.actor}}@users.noreply.github.com>
+  USERPROFILE: "${{github.workspace}}\\home"
+  BUILD_ONLY: "${{github.event.client_payload.build_only}}"
+  REPOSITORY: "${{github.event.client_payload.repository}}"
+  REF: "${{github.event.client_payload.ref}}"
+
+jobs:
+  bundle-artifacts:
+    runs-on: windows-latest
+    steps:
+      - name: Download git-sdk-64-build-installers
+        shell: bash
+        run: |
+          a=git-sdk-64-build-installers && mkdir -p $a && curl -# https://wingit.blob.core.windows.net/ci-artifacts/$a.tar.xz | tar -C $a -xJf - &&
+          mkdir -p home &&
+          git config --global windows.sdk32.path '' &&
+          git config --global windows.sdk64.path "$PWD/$a"
+      - name: Clone build-extra
+        run: git clone --single-branch -b master https://github.com/git-for-windows/build-extra git-sdk-64-build-installers\usr\src\build-extra
+      - name: Prepare home directory for GPG signing
+        if: env.GPGKEY != '' && env.PRIVGPGKEY != ''
+        shell: bash
+        run: |
+          echo '$PRIVGPGKEY' | tr % '\\n' | gpg $env:GPG_OPTIONS --import &&
+          mkdir -p home &&
+          git config --global gpg.program "$PWD/git-sdk-64/usr/src/build-extra/gnupg-with-gpgkey.sh"
```

Comment:
`mkdir` was done in the previous step?

Should `git-sdk-64/usr` be `git-sdk-64-build-installers/usr`?